help/en/html/hardware/can/cbus/index.shtml

```diff
@@ -388,13 +388,64 @@
         MERG CBUS + JMRI for route setting, signalling and track occupancy purposes.
         </p>
       
-        <p>JMRI will send the DCC events over the MERG CBUS network and can be monitored within the MERG CBUS console</p>
+        <p>JMRI will send and receive the DCC instruction messages over the MERG CBUS network
+        and can be monitored within the MERG CBUS console</p>
 
         <p>Loco sessions are best viewed with the 
         <a href="../../../../package/jmri/jmrix/can/cbus/swing/cbusslotmonitor/CbusSlotMonitorPane.shtml" 
          >CBUS Command Station Monitor</a>, 
         which updates the sessions in a table format.</p>
+        
+        <p>There are a number of DC Command Stations which have been designed by MERG members
+        which use the standard CBUS protocol for requesting and controlling loco sessions.</p>
+
+        <h3>Throttles</h3>
+        
+        
+        <p>All of the programs within the JMRI suite ( including the JMRI WiThrottle server )
+        internally share the same command station slot for a given loco address,
+        so all of JMRI appears to the command station as a single CANCAB for any single address.</p>
+
+        <p>JMRI ( via the CBUS Node Manager ) can tell if a command station is available which 
+        supports the CBUS steal or share feature ( eg. supported with CANCMD v4 + )</p>
+        
+        <p>The Steal / Share feature is for when throttles are requested which are already in use 
+        by something external to JMRI.</p>
+        
+        <p>When a JMRI application requests a loco ( which is already in a CBUS session ), 
+        JMRI will check the main <a 
+        href="../../tools/throttle/ThrottlePreferencesPane.shtml">Throttles Preferences</a>.</p>
+        
+        <p>If Silent Steal or Silent Share has been selected, this will be the default action.</p>
+        
+        <p>If neither have been checked and you are using the main JMRI Throttle, you will be asked
+        if you want to Steal, Share, or cancel.</p>
+        
+        <img src="images/web/cbus-throttle-steal-share-dialogue-278x128.png"
+        width="278" height="128" alt="JMRI MERG CBUS Steal Share Dialogue">
+        
+        <p>If you are using an automation script and neither option is checked, the system will
+        attempt to acquire a Share on the session.</p>
+        
+        <p>The command station may disable or enable steal / share at any point, 
+        so even if you have a preferred option, an automation script may attempt to use
+        the non-preferred option to obtain the session.</p>
+        
+        <p>When a Throttle has been stolen from JMRI, the single JMRI throttle window for that loco 
+        ( if open ) returns to address selection mode, ie throttle slider and function keys greyed out.</p>
```

Comment:
@rhwood what do the HIG guidelines say we should do if some external action forces us to loose control.  My gut reaction is an informational message should be displayed in a dialog.  

Just going back to the no address selected state seems uninformative.

Reply:
I'll re-write this bit in the help, I realised the UI issue and re-wrote as
After running a headless check
Warning dialogue with checkbox to disable for rest of session.